apache/couchdb-fauxton
# Show error message if it can't retrieve ddoc metadata

## Overview

If Fauxton fails to load the design doc metadata, the page continues to show the Loading icon, and no error is displayed.

## Testing recommendations

You can simulate a network error by blocking the network request. 
In Chrome:
 - Open dev tools
 - Navigate to the design doc metadata page
 - Go to the dev tools, network tab. Then right click the request `/YOUR_DB/_design/YOUR_DDOC/_info`, and select 'Block request URL'.
 - Reload the page. An error message should be displayed. The loading icon is not shown.

## Checklist
